buildpipeline/pipeline.json

```diff
@@ -137,6 +137,28 @@
           }
         }
       ]
+    },
+    {
+      "Name": "Sign And Publish Packages",
```

Comment:
Is your plan to stop the publishing from the individual legs now as just do it here? I think that would be a good idea but I didn't see anything obvious that was doing that.

Reply:
No, we still need the individual legs to publish to Azure in order for this new job to find the packages to sign. I was under the impression that the plan from our old email thread was to just have the signed packages available in Azure, and publish to nuget from there

Reply:
Or do you think we should disable myget publishing by the individual legs, and have this job do the myget publishing instead?

Reply:
I realize they need to publish to azure but IIRC what happens today is whatever leg finishes last publishes to myget and given you are creating a sync point now we should be able to stop doing that and just have your leg publish to myget.

Reply:
Fair point, I'll work on removing the myget publish from the Finalize step & adding it to the new pipeline job.